tests/data_files/base64/def_b64_ff.txt

```diff
@@ -0,0 +1,5 @@
+// Ensure that the b64 parser continues after encountering a 0xFF
+// character.  Note that this byte is invalid UTF-8, making this
+// entire file invalid UTF-8.  Use care when editing.
```

Comment:
It might be better to rename this file to have a non-text extension (e.g. `.bin`). In addition to being a signal to people who would edit the file, checking the encoding of text files is something we might want to add to `check_files.py`.